### DISCUSSION TITLE
Align q-card-actions to bottom of Fit Card

### DISCUSSION THREAD

**Discussion by dallasbeek** [quasarframework/quasar#17564] [General - Components / Directives / etc]
This seems like it should be simple but can't seem to find the right settings.
I have cards on a page with class="q-pa-xs fit"

I simply want to align the actions to the bottom of the cards so when the cards appear horizontally the buttons align at the bottom.  See screen shot.

![Cards-10-07-2024_06_02_PM](https://github.com/user-attachments/assets/e616f678-25dd-447f-b6d3-c0e566d7e220)

**Comment by paladyne-io** [comment]
Thanks to Chat GPT :-) 

```
<q-card-actions align="right" class="actions">

<style scoped>
.actions {
  margin-top: auto; /* Push actions to the bottom */
}
</style>
```
